src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -320,19 +320,45 @@ public boolean conferenceRequest(
         room = room.toLowerCase();
 
         JitsiMeetConferenceImpl conference;
+        String id;
         synchronized (conferencesSyncRoot)
         {
-            if (!conferences.containsKey(room))
+            conference = conferences.get(room);
+            if (conference == null)
+            {
+                id = generateConferenceId();
```

Comment:
generateConferenceId() needs the lock, but we specifically want createConference to execute without holding the lock.